tests/unit/select_test.coffee

```diff
@@ -116,6 +116,62 @@ test 'Test keyboard interaction', ->
       pressing ESC')
     validateFocus('Select component should be focused after pressing ESC')
 
+test 'Test click interaction', ->
+  expect 3
+
+  select = @subject
+    content: ['foo', 'bar', 'barca', 'baz']
+  @append()
+
+  selectComponent = select.$()
+
+  validateDropdownVisible = (messageVisible) ->
+    ok isVisible(find '.ember-select-results', selectComponent),
```

Comment:
It was there before we shaped the way we write tests. I will remove all of them when I refactor the tests here.